os_migrate/localhost_inventory.yml

```diff
@@ -1,5 +0,0 @@
-migrator:
-  hosts:
-    localhost:
-      ansible_connection: local
-      ansible_python_interpreter: "{{ ansible_playbook_python }}"
```

Comment:
My only concern https://github.com/os-migrate/os-migrate/commit/c6070287f57ca4383a945a54eb03a4b8acac37ec

How do we have variables in the inventory files? How does this gets populated?